src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/actions/manual/ManualTriggerAction/index.jelly

```diff
@@ -3,7 +3,7 @@
     <l:layout title="${%Gerrit Manual trigger}" norefresh="true" permission="${it.requiredPermission}">
         <l:header>
             <script src="${rootURL}${it.getJsUrl('gerrit-search.js')}" type="text/javascript"/>
-          <link rel="stylesheet" href="${rootURL}/plugin/gerrit-trigger/css/gerrit.css" type="text/css"/>
+            <link rel="stylesheet" href="${rootURL}/plugin/gerrit-trigger/css/gerrit.css" type="text/css"/>
         </l:header>
```

Comment:
Is this relevant?

Reply:
it caused problems with using dark theme. Also to be conform with CSP, there should be no inline style tags